renovate.json

```diff
@@ -46,6 +47,19 @@
       "groupName": "Pipeline dependencies"
     }
   ],
+  "customManagers": [
+    {
+      "description": "Cs files manager for docker image versions (not built-in in Renovate)",
```

Comment:
Could we add a bit more details, as I understand it's to update docker image version that are written in .cs files?